omod/src/main/webapp/pages/showPatientInfo.gsp

```diff
@@ -49,10 +289,16 @@
             <div class="col2" align="left" style="display:inline-block; width: 150px"><b>Age:</b></div>
             <div class="col2" align="left" style="display:inline-block; width: 150px""><span id="age"></span>${age}
         </div>
+        <div class="onerow" align="left">
+            <div class="col2" align="left" style="display:inline-block; width: 150px"><b>Gender:</b></div>
+            <div class="col2" align="left" style="display:inline-block; width: 150px""><span id="age"></span>${gender}
+        </div>
     <div class="onerow" align="left">
             <div class="col2" align="left" style="display:inline-block; width: 150px"><b>You were Served by:</b></div>
             <div class="col2" align="left" style="display:inline-block; width: 150px""><span id="user"></span>${user}
         </div>
     </div>
-
-</div>
\ No newline at end of file
+</div>
+<button onclick="window.print()">Print Receipt</button>
```

Comment:
After successful printing of the receipt, the page need to redirect to **http://41.89.94.100:8080/openmrs/initialpatientqueueapp/patientQueueHome.page** that need to happen in the controler, plus we need all the required fileds be included on the receipt